lib/src/fileset.rs

```diff
@@ -159,6 +161,47 @@ impl FilePattern {
         Self::file_glob_at(dir, pattern, true)
     }
 
+    /// Pattern that matches files with the given extension recursively from
+    /// cwd.
+    pub fn cwd_extension_glob(
+        path_converter: &RepoPathUiConverter,
+        extension: impl AsRef<str>,
+    ) -> Result<Self, FilePatternParseError> {
+        let ext_glob = Self::format_extension_glob(extension)?;
+        Self::cwd_file_glob(path_converter, ext_glob)
```

Comment:
nit: it's probably better to construct `FileGlob { .. }` without using `cwd_file_glob()`, etc. It doesn't make sense to split machine-generated glob pattern and parse it as a user expression. There should be no syntax error to report.